src/UI/Component/Panel/Factory.php

```diff
@@ -19,8 +19,10 @@ interface Factory
      *   purpose: >
      *      Standard Panels are used in the center content section to group content.
      *   composition: >
-     *      Standard Panels consist of a title and a content section. The
-     *      structure of this content might be varying from Standard
+     *      Standard Panels consist of a content and a top section.
+     *      The top section may contain a title and a sortation view control.
+     *      The content section may contain a Pagination view controller or a Section view controller.
```

Comment:
... Pagination View Control or a Section View Control.

Reply:
Removed due to feedback from @klees